Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -48,11 +48,22 @@ - (void)identify:(SEGIdentifyPayload *)payload
 {
     [self.amplitude setUserId:payload.userId];
     [self.amplitude setUserProperties:payload.traits];
+    NSDictionary *groups = payload.integrations[@"Amplitude"][@"groups"];
```

Comment:
For this line what happens if a user sends `Amplitude: false` instead of a dictionary? Depending on the strictness, it would evaluate to `false[@"groups"]` which might crash.

e.g. `[self.analytics track:@"event" properties:@{} options:{ @"integrations" : @{ @"Amplitude" : false } }`.